src/intersection/README.md

```diff
@@ -1,34 +1,82 @@
-# Intersection
+# Intersection: Creates an array of values that are in two arrays
 
-Write a function that finds all the values shared between two arrays.
+<!-- BEGIN DOCS -->
+
+<a name="arrayIntersection"></a>
+
+## arrayIntersection ⇒ <code>Array</code>
+
+Creates an array of values that are in both the first and the second arrays.
+
+**Returns**: <code>Array</code> - Returns the new array of filtered values.
+
+| Param    | Type               | Default         | Description            |
+| -------- | ------------------ | --------------- | ---------------------- |
+| [array]  | <code>Array</code> | <code>[]</code> | The array to inspect.  |
+| [values] | <code>Array</code> | <code>[]</code> | The values to exclude. |
+
+> Docs generated: Tue May 18 2021, 11:09:46 AM
+
+<!-- END DOCS -->
 
 ---
 
-## Docstring
+## Strategy
+
+I chose to used built-in array and object methods to solve this challenge, because once you know them they are easy to read and save a lot of lines and time.
+
+---
+
+## Implementation
+
+I use an `if` and `else` statement:
```

Comment:
Language does not look professional, instead of using first or second person, using passive voice looks more professional.